.changeset/curly-onions-tie.md

```diff
@@ -0,0 +1,18 @@
+---
+'@twilio-paste/disclosure': patch
+'@twilio-paste/form-pill-group': patch
+'@twilio-paste/menu': patch
+'@twilio-paste/modal': patch
+'@twilio-paste/popover': patch
+'@twilio-paste/tabs': patch
+'@twilio-paste/tooltip': patch
+'@twilio-paste/core': patch
+'@twilio-paste/disclosure-primitive': patch
+'@twilio-paste/menu-primitive': patch
+'@twilio-paste/modal-dialog-primitive': patch
+'@twilio-paste/non-modal-dialog-primitive': patch
+'@twilio-paste/tabs-primitive': patch
+'@twilio-paste/tooltip-primitive': patch
+---
+
+Made a slight improvement to the TypeScript typings of several packages for better interoperability.
```

Comment:
Question: Are these files still changed? I don't see them affected.

Reply:
This is tricky because although we dont change the code, we change the tsconfig, which causes a different outcome in the built `dist` files. In order to ship the changes, I have to give them a changeset.

Reply:
ahhhhhh ok!